src/Sylius/Bundle/ShopBundle/Resources/views/Cart/Summary/_item.html.twig

```diff
@@ -1,12 +1,16 @@
 {% import "@SyliusShop/Common/Macro/money.html.twig" as money %}
 
 {% set product_variant = item.variant %}
+{% set pricing_for_active_channel = product_variant.getChannelPricingForChannel(cart.channel) %}
 
 <tr {{ sylius_test_html_attribute('cart-product-row', item.productName) }}>
     <td class="single line" {{ sylius_test_html_attribute('cart-item', loop_index|default(null) ) }}>
         {% include '@SyliusShop/Product/_info.html.twig' with {'variant': product_variant} %}
     </td>
     <td class="right aligned">
+        {% if pricing_for_active_channel.originalPrice is not null %}
```

Comment:
You should check here also if there is a catalog promotion applied (I'm not sure because the administrator can set also the original price manually) or if the original price is greater than unit price

Reply:
as you said admin also can change the original price, so I do not see a reason to check if CP is applied, I can check If the original price is greater than the unit price but IMO this check should be validated on `originalPrice` input in the admin Panel

Reply:
I partially agree, but displaying the crossed-out original price only makes sense if it is greater than the current price. For now, if I'm correct, we don't have validation for the original price, so we should cover it here, especially in the case when the original price === price, what is a correct situation but has no sense to crossed-out it here.